presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -3030,6 +3034,19 @@ public void commit()
         metastore.commit();
     }
 
+    @Override
+    public PartitioningFilterCoverage getPartitioningFilterCoverage(ConnectorTableLayoutHandle connectorTableLayoutHandle, Set<String> relevantPartitionColumns)
+    {
+        HiveTableLayoutHandle tableHandle = (HiveTableLayoutHandle) connectorTableLayoutHandle;
+        List<String> partitionKey = tableHandle.getPartitionColumns().stream().map(HiveColumnHandle::getName).collect(toList());
```

Comment:
nit: I think we usually do `toImmutableList()` and `toImmutableSet()` whenever feasible. 

Also I personally prefer to put new lines for stream APIs, e.g. 

```java
            List<String> partitionKey = tableHandle.getPartitionColumns().stream()
                    .map(HiveColumnHandle::getName)
                    .collect(toImmutableList());
```

Reply:
Actually I think we don't need this `partitionKey` anymore (we can directly compute `relevantColumns` )